<!--
* Copyright (c), 2014
* - Alexandre Jumeline
* - Perrine Fauconnier
* All rights reserved.
*
* This file is part of Warhole.
*
* Warhole is free software: you can redistribute it and/or modify
* it under the terms of the GNU General Public License as published by
* the Free Software Foundation, either version 3 of the License, or
* any later version.
*
* Warhole is distributed in the hope that it will be useful,
* but WITHOUT ANY WARRANTY; without even the implied warranty of
* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
* GNU General Public License for more details.
*
* You should have received a copy of the GNU General Public License
* along with Warhole. If not, see <http://www.gnu.org/licenses/>.
 -->

<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerrainWindow</class>
 <widget class="QDialog" name="TerrainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Décors</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Choisissez un type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboType">
             <item>
              <property name="text">
               <string>&lt;aucune&gt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Forêt</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Rivière</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Obstacle</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bâtiment</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Colline</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Falaise</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Marécage</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Autre</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Choisissez un type de couvert</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboCover">
             <item>
              <property name="text">
               <string>&lt;aucune&gt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Pas de couvert</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Couvert Léger</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Couvert Lourd</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Nom : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QLabel" name="label_22">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Image du Terrain</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <item>
                <widget class="QLineEdit" name="lineEditImage"/>
               </item>
               <item>
                <widget class="QToolButton" name="toolButtonImage">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxSight">
           <property name="text">
            <string>Bloque les lignes de vue</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxDangerous">
           <property name="text">
            <string>Terrain dangereux</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxUnsurmountable">
           <property name="text">
            <string>Terrain Infranchissable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Modifier la taille</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Hauteur (en pas)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Largeur (en pas)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>WhatsThisCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Ici sont décrites les règles spéciale de la figurine, comme la peur, la terreur, l'immunité à la psychologie...</string>
             </property>
             <property name="text">
              <string>Règles spéciales supplémentaires</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="toolTip">
          <string>Ajoute le régiment en cours d'édition à l'armée.</string>
         </property>
         <property name="text">
          <string>ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="toolTip">
          <string>Retire le régiment sélectionné de l'armée.</string>
         </property>
         <property name="text">
          <string>supprimer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editButton">
         <property name="toolTip">
          <string>Permet d'éditer les paramètre du régiment sélectionné dans l'armée.</string>
         </property>
         <property name="text">
          <string>éditer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Terrains existants :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeViewExistingTerrains"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
